src/domains/transaction/components/SendTransactionForm/SendTransactionForm.tsx

```diff
@@ -112,8 +127,10 @@ export const SendTransactionForm = ({ children, networks, profile, transactionTy
 					step={0.01}
 					onChange={(currency: { display: string; value: string }) => {
 						setValue("fee", currency, { shouldValidate: true, shouldDirty: true });
+						validateFee();
 					}}
 				/>
+				{feeWarning && <FormHelperText isWarning errorMessage={feeWarning} />}
```

Comment:
This will override error validations like the `required` rule. You may need to add a custom `warningMessage` prop.